lib/init/rc.shutdown

```diff
@@ -46,4 +50,14 @@ main() {
     }
 }
 
+set +f
```

Comment:
This can be removed as we don't disable globbing in `rc.shutdown`. I may even remove the globbing restriction from `rc.boot`.

Reply:
Thank you. This is what I wanted originally.